pkg/ucauth/kratos/sms.request.jsonnet

```diff
@@ -0,0 +1,20 @@
+
+local templateCode = 'sms_123';
+local regionId = 'cn-hangzhou';
+local assessKeyId = 'foo';
```

Comment:
I see this file does not really work, is it just an example? And how does it work in production?

Reply:
As aliyuncs OpenAPI requires signature, I pass signature info as external variables in jsonnet from global config. (It can be called with jsonnet foo.jsonnet --ext-str foo=bar). I construct the string to be signed locally with aliyun Go SDK and source code change. I will figure out another way instead.